src/Schema/PostgreSQLSchemaManager.php

```diff
@@ -93,36 +71,34 @@ public function getSchemaSearchPaths(): array
     }
 
     /**
-     * Gets names of all existing schemas in the current users search path.
-     *
-     * This is a PostgreSQL only function.
-     *
-     * @internal The method should be only used from within the PostgreSQLSchemaManager class hierarchy.
+     * Gets names of all existing schemas in the current user's search path.
      *
      * @return array<int, string>
+     *
+     * @throws Exception
      */
-    public function getExistingSchemaSearchPaths(): array
+    protected function getExistingSchemaSearchPaths(): array
```

Comment:
Should these also become final?

Reply:
The purpose of this API is not really clear to me: there's a single use case which uses only the first element of the path: https://github.com/doctrine/dbal/blob/d19875a0f127c1cbeb77e160619ec36e096dc8d3/src/Schema/PostgreSQLSchemaManager.php#L177-L180
So I'd expect some more rework here in the future.

But yeah, `final` makes sense regardless.